code/modules/mob/living/carbon/life.dm

```diff
@@ -723,6 +723,37 @@ All effects don't start immediately, but rather get worse over time; the rate is
 
 	return fullness
 
+/mob/living/carbon/has_reagent(reagent, amount = -1, needs_metabolizing = FALSE)
+	. = ..()
+	if(.)
+		return
+	var/obj/item/organ/stomach/belly = getorganslot(ORGAN_SLOT_STOMACH)
+	if(!belly)
+		return FALSE
+	. = belly.reagents.has_reagent(reagent, amount, needs_metabolizing)
+
+/mob/living/carbon/remove_reagent(reagent, amount=-1, safety)
+	if(amount == -1)
```

Comment:
Can you turn this `-1` into a define, to avoid magical numbers?
This should be documented on the base proc.

Reply:
nulling it is better